docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1016,6 +1016,104 @@ table my_ipv4_lkup {
   ...
 }
 ~ End p4example
+### Custom Annotations in P4Info
```

Comment:
can we add an empty tine before the section?

Reply:
Done